chain/mining.go

```diff
@@ -174,7 +174,7 @@ func (bm *BuiltinMining) BuildBlock(ctx context.Context, height uint32, chordID
 				TransactionsRoot: txnRoot.ToArray(),
 				WinnerHash:       winnerHash.ToArray(),
 				WinnerType:       winnerType,
-				SignerPk:         bm.account.PublicKey.EncodePoint(),
+				SignerPk:         bm.account.PublicKey,
```

Comment:
Will this change cause hard fork?

Reply:
It won't, the previous `PublicKey.EncodePoint()` is the same as `PublicKey` now, both are the bytes of pk